tensorflow/python/eager/wrap_function.py

```diff
@@ -308,6 +334,9 @@ def _fetch_preprocesing_callback(f):
     pruned_graph.variables = self.graph.variables
 
     def _structured_output_mapping(fetched):
+      """`nest.map_structure()` callback."""
+      if isinstance(fetched, sparse_tensor.SparseTensor):
```

Comment:
`map_structure` actually has an expand_composites option too. I think if we use that we can get rid of this special case? So inside the map we'll just get regular Tensors.

Reply:
+1.  We should be able to just change:

    pruned_graph.structured_outputs = nest.map_structure(
        _structured_output_mapping, fetches)

to:

    pruned_graph.structured_outputs = nest.map_structure(
        _structured_output_mapping, fetches, expand_composites=True)

and leave _structured_output_mapping unchanged.  That way, it will work for all composite tensors.

Reply:
Making that change now.